packages/snjs/lib/protocol/payloads/deltas/remote_rejected.ts

```diff
@@ -0,0 +1,33 @@
+import { CreateSourcedPayloadFromObject } from '@Payloads/generator';
+import { PayloadsDelta } from '@Payloads/deltas/delta';
+import { PayloadSource } from '@Payloads/sources';
+import { ImmutablePayloadCollection } from '@Protocol/collection/payload_collection';
+
+export class DeltaRemoteRejected extends PayloadsDelta {
+  // eslint-disable-next-line @typescript-eslint/require-await
+  public async resultingCollection(): Promise<ImmutablePayloadCollection> {
```

Comment:
Why do we want this to be async?

Reply:
It's overriding parent function which is async. Removing async in subclass implementation results in a warning. So only way was to add the eslint-disable rule.